features/srv_mainpage.feature

```diff
@@ -61,6 +61,22 @@ Feature: Main landing page options and preferences
     And I should see a "Configuration" link
```

Comment:
on line https://github.com/SUSE/spacewalk-testsuite-base/blob/36a69b8557c98903211138ff9cfb70fc1fcd4843/features/srv_mainpage.feature#L52

Could you rename  the scenarion
''' Main Menu as non Administrator'''

Reply:
I went for `Main menu as regular user`.

Reply:
good

Reply:
LGTM